.travis.yml

```diff
@@ -8,6 +8,8 @@ sudo: false
 before_install:
    - export PATH=$HOME/gopath/bin:$PATH
 
+install: "pip install --user -r requirements.txt"
```

Comment:
perhaps, instead of using `pip`, one could use `yum` to install `setuptools` and `cffi` ?